apps/api/src/app/message-template/usecases/find-message-templates-by-layout/find-message-templates-by-layout.use-case.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { MessageTemplateEntity, MessageTemplateRepository } from '@novu/dal';
+
+import { FindMessageTemplatesByLayoutCommand } from './find-message-templates-by-layout.command';
+
+const DEFAULT_PAGE_SIZE = 100;
+
+@Injectable()
+export class FindMessageTemplatesByLayoutUseCase {
+  constructor(private messageTemplateRepository: MessageTemplateRepository) {}
+
+  async execute(command: FindMessageTemplatesByLayoutCommand): Promise<MessageTemplateEntity[]> {
```

Comment:
I really like all these separations. You think ahead💪🏻

I see multiple possible future uses for this